RecoBTag/TrackProbability/test/CalibrationSkeleton.cc

```diff
@@ -179,109 +165,83 @@ class CalibrationSkeleton : public edm::EDAnalyzer {
 //
 // constructors and destructor
 //
-CalibrationSkeleton::CalibrationSkeleton(const edm::ParameterSet& parameters)
-{
-}
-
-void
-CalibrationSkeleton::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
-{
-
-
+CalibrationSkeleton::CalibrationSkeleton(const edm::ParameterSet& parameters) {}
 
+void CalibrationSkeleton::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup) {
   using namespace edm;
   using namespace reco;
   using namespace std;
- 
+
   Handle<reco::VertexCollection> primaryVertex;
-  iEvent.getByLabel("offlinePrimaryVertices",primaryVertex);
-  
-  //*********************************************************************************** 
-  //look at reco vertices 
-  const  reco::Vertex  *pv;
+  iEvent.getByLabel("offlinePrimaryVertices", primaryVertex);
+
+  //***********************************************************************************
+  //look at reco vertices
+  const reco::Vertex* pv;
   bool newvertex = false;
-  
+
   bool pvFound = (primaryVertex->size() != 0);
-  if(pvFound)
-    {
-      pv = &(*primaryVertex->begin());
-    }
-  else 
-    { 
-      reco::Vertex::Error e;
-      e(0,0)=0.0015*0.0015;
-      e(1,1)=0.0015*0.0015;
-      e(2,2)=15.*15.;
-      reco::Vertex::Point p(0,0,0);
-      pv=  new reco::Vertex(p,e,1,1,1);
-      newvertex=true;
-    }
-  
+  if (pvFound) {
+    pv = &(*primaryVertex->begin());
+  } else {
+    reco::Vertex::Error e;
+    e(0, 0) = 0.0015 * 0.0015;
+    e(1, 1) = 0.0015 * 0.0015;
+    e(2, 2) = 15. * 15.;
+    reco::Vertex::Point p(0, 0, 0);
+    pv = new reco::Vertex(p, e, 1, 1, 1);
+    newvertex = true;
+  }
+
   //*************************************************************************
   //look at JetTracks
   edm::Handle<JetTracksAssociationCollection> associationHandle;
   iEvent.getByLabel("ak5JetTracksAssociatorAtVertex", associationHandle);
   reco::JetTracksAssociationCollection::const_iterator it = associationHandle->begin();
-  
-  
-  
+
   //***********************************************************************************
   //mandatory for ip significance reco
-  const TransientTrackBuilder * m_transientTrackBuilder_producer;
+  const TransientTrackBuilder* m_transientTrackBuilder_producer;
   edm::ESHandle<TransientTrackBuilder> builder;
-  iSetup.get<TransientTrackRecord>().get("TransientTrackBuilder",builder);
+  iSetup.get<TransientTrackRecord>().get("TransientTrackBuilder", builder);
   m_transientTrackBuilder_producer = builder.product();
```

Comment:
```suggestion
auto const m_transientTrackBuilder_producer = &iSetup.getData(transientTrackBuilderToken_)
```
and you'll need to define `transientTrackBuilderToken_` in the header as
``` 
const edm::ESGetToken<TransientTrackBuilder, TransientTrackRecord> transientTrackBuilderToken_;
```

Reply:
implemented as suggested above